sources/packages/backend/apps/db-migrations/src/sql/Applications/Add-col-current-processing-assessment-id.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    sims.applications
+ADD
+    COLUMN current_processing_assessment_id INT REFERENCES sims.student_assessments(id);
```

Comment:
I can understand after your explanation why we do not need to have a copy the default current_assessment to this id.